### ISSUE TITLE
Display of Table Formula and TableFormulaWithXArguments

### ISSUE THREAD

**Issue by msevestre** [Open-Systems-Pharmacology/MoBi#1040]
@PavelBal @rwmcintosh @Yuri05 
We have them now in Expression Profile and in the simulation

see attached project 
[Ontogeny_factor.zip](https://github.com/Open-Systems-Pharmacology/MoBi/files/11763293/Ontogeny_factor.zip)

Thsi is in the simulation
![image](https://github.com/Open-Systems-Pharmacology/MoBi/assets/1041237/9ddacf6f-86ce-4db4-a83a-6b411ff5b4fa)

This is in the expression profile
![image](https://github.com/Open-Systems-Pharmacology/MoBi/assets/1041237/77d3f927-45d6-4477-898c-48aaedee829f)

In short. We have no idea what's going on. The values are correct but it's a bit..useless

**Comment by msevestre** [comment]
![image](https://github.com/Open-Systems-Pharmacology/MoBi/assets/1041237/7a2ff0e8-c24e-4ba0-9bf5-43e47556d931)

**Comment by PavelBal** [comment]
The best way would probably be - by clicking on the cell,  a view of the table opens. This is probably not easy to implement without sacrificing the drop-down for formula selection.

Alternative - only show the name of the formula, and the user must go to the "Formulas" tab to view it.

**Comment by Yuri05** [comment]
Table **with argument** parameters (`Ontogeny factor` and `Ontogeny factor GI` obviously cannot be displayed and showing `<Not Available>` is OK.
For the **table** parameters: would prefer the same way as in PK-Sim (e.g. for pH-Solubility table in compound):
* show the button "Edit Table"
![grafik](https://github.com/Open-Systems-Pharmacology/MoBi/assets/25061876/9034b694-3450-416e-a45f-43e93af971ca)
* When clicking on the button: open separate window where the table values can be modified (or imported)
![grafik](https://github.com/Open-Systems-Pharmacology/MoBi/assets/25061876/fdd838e5-5bec-426d-b678-e88596b91389)

**Comment by PavelBal** [comment]
Update: The formula with argument is located in the molecule, the expression profiel only holds the table formula. However, these are not displayed (selecting a table formula in the "Formulas" tab crashes:

Application:
MoBi® 12.0.75

Formula 'DistributedTableFormula' not supported at the moment

Stack trace:
```
at MoBi.Presentation.Presenter.FormulaPresenterCache.PresenterFor(Type formulaType)
   at MoBi.Presentation.Presenter.FormulaPresenterCache.PresenterFor(IFormula formula)
   at MoBi.Presentation.Presenter.FormulaCachePresenter.Select(IFormula formula)
   at MoBi.Presentation.Presenter.FormulaCachePresenter.Select(FormulaBuilderDTO formulaDTO)
   at MoBi.UI.Views.FormulaCacheView.<>c__DisplayClass4_1.<InitializeBinding>b__7()
   at OSPSuite.UI.Controls.BaseUserControl.<>c__DisplayClass23_0.<OnEvent>b__0()
   at OSPSuite.Utility.Extensions.ExceptionExtensions.DoWithinExceptionHandler(Object callerObject, Action actionToExecute)
```


![image](https://github.com/Open-Systems-Pharmacology/MoBi/assets/25903040/d6f3c253-a35d-4a4a-a4bf-3cb4b20218c9)


proposed solution: in the parameters list, show the name of the formula, which can be viewed in the "Formulas"-tab.

**Comment by rwmcintosh** [comment]
This is going to be closed as nothing left to do after #1396 and #1395 are done.
